package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "root",
 	"private": true,
+	"files": [],
```

Comment:
According to the docs this is a whitelist of files to publish - why have you set it, and why is it blank?

Reply:
This was the fix I found for [node/no-unpublished-require](https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-require.md) on line 11 of clean.js
What is needed is for clean.js to be excluded from publishing, which can be done with either the files property or an npmignore file.
Since the root package.json doesnt describe an actual package and is marked as private (which prevent publishing) it is safe to have it list no files. This method means we do not need to maintain an npmignore file if more helper scripts are added at a later date.